### PR TITLE
Move models package into functions folder

### PR DESCRIPTION
**Pull request by pauljohanneskraft**
# Move models package into functions folder

## :recycle: Current situation & Problem
Apparently when functions are deployed, they only have access to the functions directory, so moving the models package into that folder should solve the issue.

## :gear: Release Notes 
*Add a bullet point list summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented or links to the documentation if it appends or changes the public interface.*

## :books: Documentation
*Please ensure that you properly document any additions in conformance to [Spezi Documentation Guide](https://github.com/StanfordSpezi/.github/blob/main/DOCUMENTATIONGUIDE.md).*
*You can use this section to describe your solution, but we encourage contributors to document your reasoning and changes using in-line documentation.* 

## :white_check_mark: Testing
*Please ensure that the PR meets the testing requirements set by CodeCov and that new functionality is appropriately tested.*
*This section describes important information about the tests and why some elements might not be testable.*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordBDHG/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordBDHG/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordBDHG/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordBDHG/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pauljohanneskraft** [comment]
It was apparently an issue with node 22.7.0... Reverting to 22.6.0 solved the issue.

File: .github/workflows/lintbuildandtest.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
         run: npm run lint:strict
       - name: Test
         id: test
-        run: firebase emulators:exec --only auth,firestore 'npm --prefix functions run test:ci'
+        run: npm run test:ci
```
Comment (PSchmiedmayer) [inline .github/workflows/lintbuildandtest.yml:38]: Wondering if we should re-enable the tests with the emulator now that we have identified the fix?

**Review by PSchmiedmayer — APPROVED** [review]
Thank you for fixing this; only thing that would be great would be to re-enable the testing with the emulator if we use it in the current set of tests.

File: .github/workflows/lintbuildandtest.yml (hunk 1 above)
Comment (pauljohanneskraft) [reply .github/workflows/lintbuildandtest.yml:38]: This is running with the emulators - I just moved this part into the outermost package.json as a script

**Pull request merged**